documentation/modules/configuring/con-config-mirrormaker2.adoc

```diff
@@ -207,11 +207,11 @@ spec:
 <4> Specification for the Kafka clusters being synchronized.
 <5> Cluster alias for the source Kafka cluster.
 <6> Authentication for the source cluster, specified as mTLS, token-based OAuth, SASL-based SCRAM-SHA-256/SCRAM-SHA-512, or PLAIN.
-<7> Bootstrap server for connection to the source Kafka cluster.
+<7> Bootstrap server for connection to the source Kafka cluster. You can specify more than one address in case a server goes down. Each address takes the format `<cluster_name>-kafka-bootstrap:<port_number>`. The Kafka cluster doesn't need to be managed by Strimzi or deployed to a Kubernetes cluster.
```

Comment:
I would leave out the new part and just change the original from _bootstrap server_ to _bootstrap address_. The new part is confusing as it might be multiple broker addrsses, but from the same cluster.

```suggestion
<7> Bootstrap address for connection to the source Kafka cluster. The address takes the format `<cluster_name>-kafka-bootstrap:<port_number>`. The Kafka cluster doesn't need to be managed by Strimzi or deployed to a Kubernetes cluster.
```